R/pkg/R/DataFrame.R

```diff
@@ -2863,11 +2863,19 @@ setMethod("unionAll",
 #' \code{UNION ALL} and \code{UNION DISTINCT} in SQL as column positions are not taken
 #' into account. Input SparkDataFrames can have different data types in the schema.
 #'
+#' When the parameter `allowMissingColumns` is `TRUE`, this function allows
```

Comment:
Grammar revision:

> When the parameter `allowMissingColumns` is `TRUE`, the set of column names in `x` and `y` can differ; missing columns will be filled as null. Further, the missing columns of `x` will be added at the end in the schema of the union result.

Reply:
Sounds good, but for consistency it should fixed in Python and Scala as well.

Should it be 

> columns will be filled as null.

or 

> columns will be filled with null.

though?

Reply:
good catch, with null is better